programmers/난이도별/level01.정수_제곱근_판별/HyeonJeong.py

```diff
@@ -0,0 +1,8 @@
+def solution(n):
+    for i in range(1, n + 1):
+        if i * i == n:
+            return (i + 1) * (i + 1)
+        elif i == n:
+            return -1
```

Comment:
```suggestion
    for i in range(1, n + 1):
        if i ** 2 == n:
            return (i + 1) ** 2
        elif i == n:
            return -1
```

이렇게 `n ** 2`를 하면 제곱근을 구할 수 있답니다~